Source/Blazorise/Components/FocusTrap/FocusTrap.razor.cs

```diff
@@ -38,26 +38,45 @@ protected override async Task OnAfterRenderAsync( bool firstRender )
             }
         }
 
-        internal async Task SetFocus()
+        /// <summary>
+        /// Sets the focus trap.
+        /// </summary>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public async Task SetFocus()
             => await startFirstRef.FocusAsync();
 
-        private async Task FocusStartAsync( FocusEventArgs args )
+        /// <summary>
+        /// Handles the focus start event.
+        /// </summary>
+        /// <param name="args">Supplies information about a focus event that is being raised.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        protected virtual async Task OnFocusStartHandler( FocusEventArgs args )
         {
             if ( !shiftTabPressed )
             {
                 await startFirstRef.FocusAsync();
             }
         }
 
-        private async Task FocusEndAsync( FocusEventArgs args )
+        /// <summary>
+        /// Handles the focus end event.
+        /// </summary>
+        /// <param name="args">Supplies information about a focus event that is being raised.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        protected virtual async Task OnFocusEndHandler( FocusEventArgs args )
         {
             if ( shiftTabPressed )
             {
                 await endFirstRef.FocusAsync();
             }
         }
 
-        private void HandleKeyPresses( KeyboardEventArgs args )
+        /// <summary>
+        /// Handles the focus start event.
+        /// </summary>
+        /// <param name="args">Supplies information about a keyboard event that is being raised.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        protected virtual void OnKeyPressesHandler( KeyboardEventArgs args )
```

Comment:
Is the method documentation right? I don't know if it's github change that's displaying incorrectly. Seems like it has the Focus documentation instead.
Should it be `KeyPressesHandler` or `KeyPressesHandler`?

Reply:
Yep, it's a typo.

Reply:
Wait did I just made a type too when asking? haha I meant 
Should it be `KeyPressesHandler` or `KeyPressedHandler`?

Reply:
You did :D